fuse/eval/metrics/metrics_common.py

```diff
@@ -887,6 +887,23 @@ def reset(self) -> None:
         self._metric.reset()
         return super().reset()
 
+    @staticmethod
+    def _convert_tuples(ids: List[Tuple[str, int]]) -> np.ndarray:
```

Comment:
Are you assuming tuple of str and int? 
Can we do something more general?

Reply:
Thx. The method of convert is supporting all tuple of everything of any length, I need to change the interface definition